src/timemap.cpp

```diff
@@ -52,10 +52,12 @@ void Timemap::AddEntry(Object *object, GenerateTimemapParams *params)
         DurationInterface *interface = object->GetDurationInterface();
         assert(interface);
 
-        double realTimeStart = params->m_realTimeOffsetMilliseconds + interface->GetRealTimeOnsetMilliseconds();
+        double realTimeStart
+            = (int)(params->m_realTimeOffsetMilliseconds + interface->GetRealTimeOnsetMilliseconds() + .5);
```

Comment:
Instead of adding 0.5 and truncating it to an integer you should you use `std::round()` here, except you really want  `realTimeStart` and `realTimeEnd` to be integers.

Reply:
Thanks a lot @rettinghaus! I followed the suggestions by @craigsapp, but I am very happy with this formulation (see [0685034](https://github.com/rism-digital/verovio/pull/3221/commits/068503413bd604aea319686db0eec510ce886ab4))